src/System.ComponentModel.Primitives/src/System/ComponentModel/BrowsableAttribute.cs

```diff
@@ -0,0 +1,91 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.ComponentModel
+{
+    /// <devdoc>
+    ///    <para>Specifies whether a property or event should be displayed in
+    ///       a property browsing window.</para>
+    /// </devdoc>
+    [AttributeUsage(AttributeTargets.All)]
+    public sealed class BrowsableAttribute : Attribute, IIsDefaultAttribute
+    {
+        /// <devdoc>
+        ///    <para>
+        ///       Specifies that a property or event can be modified at
+        ///       design time. This <see langword='static '/>
+        ///       field is read-only.
+        ///    </para>
+        /// </devdoc>
+        public static readonly BrowsableAttribute Yes = new BrowsableAttribute(true);
+
+        /// <devdoc>
+        ///    <para>
+        ///       Specifies that a property or event cannot be modified at
+        ///       design time. This <see langword='static '/>field is read-only.
+        ///    </para>
+        /// </devdoc>
+        public static readonly BrowsableAttribute No = new BrowsableAttribute(false);
+
+        /// <devdoc>
+        /// <para>Specifies the default value for the <see cref='System.ComponentModel.BrowsableAttribute'/>,
+        ///    which is <see cref='System.ComponentModel.BrowsableAttribute.Yes'/>. This <see langword='static '/>field is read-only.</para>
+        /// </devdoc>
+        public static readonly BrowsableAttribute Default = Yes;
+
+        private bool _browsable = true;
+
+        /// <devdoc>
+        /// <para>Initializes a new instance of the <see cref='System.ComponentModel.BrowsableAttribute'/> class.</para>
+        /// </devdoc>
+        public BrowsableAttribute(bool browsable)
+        {
+            _browsable = browsable;
+        }
+
+        /// <devdoc>
+        ///    <para>
+        ///       Gets a value indicating whether an object is browsable.
+        ///    </para>
+        /// </devdoc>
+        public bool Browsable
```

Comment:
Nit: If you wanted (you don't need to), you could use C# 6 expression-bodied members syntax, e.g.
```C#
public bool Browsable => _browsable;
```